docs/getting_started_guide/eo_products_properties.rst

```diff
@@ -0,0 +1,85 @@
+.. _eo_products_properties:
+
+EO Products Properties
+======================
+
+EODAG exposes metadata for queryables based on [OGC OpenSearch Extension for Earth Observation](https://docs.ogc.org/is/13-026r9/13-026r9.html) for all providers that support it.
+All of the parameters are string based. If a parameter requires an specific format it will be defined on the list below.
+
+----------------------------
+
+* **uid**: Unique identifier for the resource, using provider specific formatting.
+* **tileIdentifier**: Tile identifier.
+* **productType**: A string identifying the entry type (e.g., ER02_SAR_IM__0P, MER_RR__1P, SM_SLC__1S, GES_DISC_AIRH3STD_V005).
```

Comment:
please use product types existing in `resources/product_types.yml` as example (`S2_MSI_L1C`, `LANDSAT_C2L2_SR`, `CAMS_EAC4`, ...)